XenAdmin/Core/Updates.cs

```diff
@@ -117,6 +171,42 @@ private static void actionCompleted(ActionBase sender)
                 lock (downloadedUpdatesLock)
                 {
                     ClientVersions = action.ClientVersions;
+                }
+            }
+
+            lock (updateAlertsLock)
+            {
+                updateAlerts.Clear();
+
+                if (succeeded)
+                {
+                    var clientUpdateAlerts = NewClientUpdateAlerts(ClientVersions, Program.Version);
+                    updateAlerts.AddRange(clientUpdateAlerts.Where(a => !a.IsDismissed()));
+
+                    var xenServerUpdateAlerts = NewXenServerVersionAlerts(XenServerVersionsForAutoCheck);

```

Comment:
Here I have made a mistake, the `xenServerUpdateAlerts` and `xenServerPatchAlerts` should be outside the `if (succeeded)` block. Same in line 222, the clientUpdateAlerts should also be outside. I can send a correction in this PR, or leave it for the next one (because I'm modifying this file for the updates from CDN), let me know.